pkg/utils/filesystem.go

```diff
@@ -34,20 +34,22 @@ import (
 )
 
 // CreateTempFile in the same directory as the binary for docker compose
-func CreateTempFile(filePath string) bool {
+func CreateTempFile(filePath string) (string, error) {
 	var _, err = os.Stat(filePath)
 
 	// create file if not exists
 	if os.IsNotExist(err) {
-		var file, err = os.Create(filePath)
-		errors.CheckErr(err, 201, "")
+		var file, createErr = os.Create(filePath)
+		if createErr != nil {
+			return "", createErr
+		}
+
 		defer file.Close()
 
 		dir, _ := os.Getwd()
-		fmt.Println("==> created file", path.Join(dir, filePath))
-		return true
+		return path.Join(dir, filePath), nil
 	}
-	return false
+	return "", nil
```

Comment:
I'm not sure what this return case means. Is it an error or does it just mean the file already existed and it's ok to re-use it? (Similar to above, can this function now get away with just returning an error or nil. Does it need to return a string?)